dev_tools/bash_scripts_test.py

```diff
@@ -88,8 +89,9 @@ def test_pytest_changed_files_file_selection():
                        'git commit -m test --quiet --no-gpg-sign\n')
     assert result.exit_code == 0
     assert result.out == 'INTERCEPTED pytest file_test.py\n'
-    assert result.err == ("Comparing against revision 'HEAD~1'.\n"
-                          "Found 1 differing files with associated tests.\n")
+    assert result.err.split() == (
```

Comment:
There are still calls to `split` present. Could you explain why these ones were kept?

Reply:
Yes, those checks are problematic because of different formatting I experience (split() is a wait to ignore whitespaces):
```
>       assert result.err == (
            "Comparing against revision 'HEAD~1'.\n"
            "Found 0 differing files with associated tests.\n")
E       AssertionError: assert 'Comparing ag...ated tests.\n' == 'Comparing aga...ated tests.\n'
E         Skipping 33 identical leading characters in diff, use -v to show
E           1'.
E         - Found        0 differing files with associated tests.
E         ?       -------
E         + Found 0 differing files with associated tests.

dev_tools/bash_scripts_test.py:81: AssertionError
```

Reply:
Oh, I didn't realize split had that behavior when you didn't give it arguments. Alright, that's fine then.